docs/user_guide/examples/explanation_interpolation.md

```diff
@@ -0,0 +1,62 @@
+# Interpolator explanation
+
+Interpolation is an important functionality of Parcels. On this page we will discuss the way it is
+implemented in **Parcels** and how to write a custom interpolator function.
+
+```{note}
+TODO: expand explanation (similar to Kernel loop explanation)
+```
+
+When we want to know the state of particles in an environmental field, such as temperature or velocity,
+we _evaluate_ the `parcels.Field` at the particles real position in time and space (`t`, `z`, `lat`, `lon`).
+In Parcels we can do this using square brackets:
+
+```
+particles.temperature = fieldset.temperature[particles.time, particles.z, particles.lat, particles.lon]
+```
+
+The values of the `temperature` field at the particles' positions are determined using an interpolation
+method. This interpolation method defines how the discretized values of the `parcels.Field` should
+relate to the value at any point within a grid cell.
+
+Each `parcels.Field` is defined on a (structured) `parcels.XGrid` or (unstructured) `parcels.UXGrid`.
+The interpolation function takes information about the particles position relative to this grid (`grid_positions`),
+as well as the `parcels.Field` values of the surrounding grid points in time and space, to calculate
```

Comment:
Not only the surrounding points; but _all_ points (in v4). This allows higher-order interpolators (using information beyond the immediate nodes around a particle)